coffeelint.json

```diff
@@ -27,5 +27,14 @@
   "braces_spacing": {
     "spaces": 0,
     "level": "error"
+  },
+  "spacing_after_comma": {
+    "level": "error"
+  },
+  "no_stand_alone_at": {
+    "level": "error"
+  },
+  "newlines_after_classes": {
+    "level": "error"
```

Comment:
I think I'm gonna also remove this one because it's requiring an extra newline after classes even if they already have a newline after their last method.

So this: 
```coffee
          class TestView
            initialize: (@model) -> this

          model = new TestModel
```

becomes:

```coffee
          class TestView
            initialize: (@model) -> this


          model = new TestModel
```